### PR TITLE
Add browser analytics

### PR DESCRIPTION
**Pull request by elevu**
This PR adds a call to Netlify passing the user agent (containing browser name and operative system) and screen size when clicking on login. This is a temporary solution to understand which browsers are being used.

**How to test**:
Go to staging, deploy this branch. When logging in you will see an http call to Netlify passing the data

**Expected outcome**:
The functionality should be working

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Removed OMIM genes panel
 - Make genes panel, pinned variants panel, causative variants panel and ClinVar panel scrollable on case page
 
+## [4.37]
+### Added
+- On login, the browser userAgent and screen size are saved anonymously to understand which browsers are used by Scout users
```
Comment (northwestwitch) [inline CHANGELOG.md:39]:
```suggestion
- On login, OS, browser userAgent and screen size are saved anonymously to understand which browsers are used by Scout users
```

Comment (northwestwitch) [reply CHANGELOG.md:39]: Is this more correct?

Comment (elevu) [reply CHANGELOG.md:39]: in the browser user agent both OS and browser information are included. Maybe I can write
`On login, OS, browser version and screen size are saved anonymously to understand how users are using Scout`

File: scout/server/blueprints/public/templates/public/index.html [hunk 2]
```diff
@@ -66,3 +66,31 @@ <h1 class="display-4">Scout</h1>
...
 </div>
 {% endblock %}
+
+{% block scripts %}
+{{ super() }}
+  <script type="text/javascript">
+    const sendBrowserInfo = () => {
+      console.log('Calling Netlify')
+      if (window.location.hostname.includes('scilifelab.se')) {
+          fetch('https://user-stats.netlify.app/.netlify/functions/add-browser', {
```
Comment (northwestwitch) [inline scout/server/blueprints/public/templates/public/index.html:76]: Does this document apply for this feature? https://www.netlify.com/gdpr-ccpa/ 

I think should be discussed with the users before merging..

File: CHANGELOG.md [hunk 3]
```diff
@@ -45,6 +45,10 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Changing the (Kind) drop-down according to (Category) drop-down in Managed variant add variant
 - Moved Gens button to individuals table
 
+## [4.37]
```
Comment (northwestwitch) [inline CHANGELOG.md:48]: We don't deploy every time we merge to master, see branching model documentation: https://github.com/Clinical-Genomics/scout/blob/master/CONTRIBUTING.md (sorry if it's not super clear!). You should branch out from current master (or pull current master into this branch) and add this features in the adds of the next release (still tagged as x.x.x). It has a long list of features

File: scout/server/blueprints/public/templates/public/index.html [hunk 4]
```diff
@@ -66,3 +66,31 @@ <h1 class="display-4">Scout</h1>
...
+              method: 'POST',
+              headers: {
+                  'Content-Type': 'application/json',
+              },
+              body: JSON.stringify({
+                  agent: navigator.userAgent,
```
Comment (northwestwitch) [inline scout/server/blueprints/public/templates/public/index.html:82]: 👍🏻 This info looks safe enough!

File: scout/server/blueprints/public/templates/public/index.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/public/templates/public/index.html:76]: Ok no, this is for when you use their software to deploy your app!

**Review by northwestwitch — APPROVED** [review]
Checked what actually is sent in the post request (I could have done it earlier!!!) and it's safe, being only userAgent, screensize and hostname.I think it's fine.

Please pull master branch into this one before merging in case it's not up-to date, see my comment in the changelog file!

File: CHANGELOG.md [hunk 5]
```diff
@@ -13,6 +13,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Display a loading icon (spinner) until the page loads completely
 - Display filter badges in cancer variants list
 - Update genes from pre-downloaded file resources
+- On login, OS, browser version and screen size are saved anonymously to understand how users are using Scout
```
Comment (northwestwitch) [inline CHANGELOG.md:16]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
Yes!

**Comment by elevu** [comment]
> Please pull master branch into this one before merging in case it's not up-to date, see my comment in the changelog file!

Updated with master right after the pr 👍 
Screen Shot 2021-07-06 at 14 24 32

**Pull request merged**